src/System.IO.Compression.Brotli/tests/CompressionStreamUnitTests.Brotli.cs

```diff
@@ -15,7 +15,12 @@ public class BrotliStreamUnitTests : CompressionStreamUnitTestBase
         public override Stream CreateStream(Stream stream, CompressionLevel level) => new BrotliStream(stream, level);
         public override Stream CreateStream(Stream stream, CompressionLevel level, bool leaveOpen) => new BrotliStream(stream, level, leaveOpen);
         public override Stream BaseStream(Stream stream) => ((BrotliStream)stream).BaseStream;
-        public override int BufferSize { get => 65520; }
+
+        // The tests are relying on an implementation detail of BrotliStream, using knowledge of its internal buffer size
+        // in various test calculations.  If the buffer size employed changes (which could also mean that ArrayPool<byte>.Shared
+        // starts giving out different array sizes), the tests will need to be tweaked.
+        public override int BufferSize => 1 << 16;
```

Comment:
Was the change from 65520 to 65536 intentional here?

Reply:
Yes.  See the comment.  This needs to match what the product src is using, as the test is doing calculations to determine, for example, what the expected compressed length of data will be.  Since we're now renting from the pool, and since the pool always snaps to power-of-2 boundaries for buffers it stores, this needed to be updated.

Reply:
> Yes. See the comment. This needs to match what the product src is using

Right. I read the comment which is why the change confused me. The product src had 65520 as the default size.

> Since we're now renting from the pool, and since the pool always snaps to power-of-2 boundaries

That's what I was forgetting. Thanks for clarifying.

Reply:
Got it.  I'll clarify the comment to stress the pool part.